src/ogma/features/reactive.ts

```diff
@@ -158,9 +158,10 @@ export class RxViz {
    * Store new node selection in state
    */
   private storeNodeSelection(state: OgmaState): OgmaState {
+    const selectedNodes = this._ogma.getNodes('all').filter((node) => node.isSelected());
```

Comment:
so many ways to do the exact same thing, could you document why this is the best and not the other?

Reply:
I kept the same method and overrode the ogma implementation. The difference is:
- Native implementation return only the visible selected nodes
- New implementation return all selected nodes including hidden ones, for example nodes that are part of a collapsed group